test/Shared/RedisServer.cs

```diff
@@ -46,7 +46,8 @@ public void Restart()
         {
             KillRedisServers();
             _server = new Process();
-            _server.StartInfo.FileName = "..\\..\\..\\..\\..\\..\\packages\\redis-64.3.0.503\\tools\\redis-server.exe";
+            string executable_path = Environment.CurrentDirectory + "\\..\\..\\..\\..\\..\\packages\\redis-64\\3.0.503\\tools\\redis-server.exe";
```

Comment:
nit: for readability, use string interpolation rather than concatenation when possible like:
`$"{Environment.CurrentDirectory}\\..\\..\\..\\..\\..\\packages\\redis-64\\3.0.503\\tools\\redis-server.exe"`